### PR TITLE
New package: tecwindow.SoundTranscriber version 1.2.0

### PR DESCRIPTION
**Pull request by MesterPerfect**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

### CODE REVIEW

**Comment by MesterPerfect** [comment]
Template: msftbot/requiresApproval/moderator

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 1:30:13 PM:  > 2023-11-21 13:30:12.312 [CLI ] Package hash verification failed. SHA256 in manifest [6bae5acd447942b94663ae4f7483ddae50a297d9989eef35e6de75081064860f] does not match download [1f2abd00ebe7d273f1ac269904064b7d73726613b510f37675df9eb7f4fbe470]

File: manifests/t/tecwindow/SoundTranscriber/1.2.0/tecwindow.SoundTranscriber.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: tecwindow.SoundTranscriber
+PackageVersion: 1.2.0
+InstallerType: inno
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/tecwindow/SoundTranscriber/releases/download/V1.2.0/SoundTranscriberV1.2.0.exe
+  InstallerSha256: 6BAE5ACD447942B94663AE4F7483DDAE50A297D9989EEF35E6DE75081064860F
```
Comment (stephengillie) [inline manifests/t/tecwindow/SoundTranscriber/1.2.0/tecwindow.SoundTranscriber.installer.yaml:10]:
```suggestion
  InstallerSha256: 1F2ABD00EBE7D273F1AC269904064B7D73726613B510F37675DF9EB7F4FBE470
```

**Comment by stephengillie** [comment]
image

File: manifests/t/tecwindow/SoundTranscriber/1.2.0/tecwindow.SoundTranscriber.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: tecwindow.SoundTranscriber
+PackageVersion: 1.2.0
+InstallerType: inno
+Installers:
```
Comment (stephengillie) [inline manifests/t/tecwindow/SoundTranscriber/1.2.0/tecwindow.SoundTranscriber.installer.yaml:7]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**